heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -379,6 +399,101 @@ export class JestPlugin implements IHeftPlugin<IJestPluginOptions> {
     ];
   }
 
+  // Resolve all specified properties using Node resolution, and replace <rootDir> with the same rootDir
+  // that we provide to Jest. Resolve if we modified since paths containing <rootDir> should be absolute.
+  private static _getJsonPathMetadata(options: IJsonPathMetadataOptions): IJsonPathMetadata {
```

Comment:
Seems like this does more than the first sentence implies.

```suggestion
  /**
   * Resolve all specified properties using Node resolution, and replace <rootDir> with the same rootDir
   * that we provide to Jest. Resolve if we modified since paths containing <rootDir> should be absolute.
   */
  private static _getJsonPathMetadata(options: IJsonPathMetadataOptions): IJsonPathMetadata {
```

What do you mean by this?:

>Resolve if we modified since paths containing <rootDir> should be absolute.